docs/adr/0013-use-eclipse-orion-as-editor.md

```diff
@@ -0,0 +1,32 @@
+# Use Eclipse Orion as editor
+Editor should provide syntax highlighting and work in browser.
+
+## Considered Alternatives
+Compared the following cloud IDEs
+
+* Eclipse Che
+* Eclipse Orion
```

Comment:
Put the chosen option always first.

Use markdown links, if possible: `[Eclipse Che](https://www.eclipse.org/che/)`. This eases readers to follow.